protocol-units/settlement/mcr/config/src/common/testing.rs

```diff
@@ -38,13 +35,17 @@ env_short_default!(
 	"0x0"
 );
 
-env_or_none!(
-    default_maybe_testing,
-    Config,
-    default_mcr_testing_admin_account_private_key,
-    default_move_token_contract_address,
-    default_movement_staking_contract_address
-);
+// env_or_none!(
+//     default_maybe_testing,
+//     Config,
+//     default_mcr_testing_admin_account_private_key,
+//     default_move_token_contract_address,
+//     default_movement_staking_contract_address
+// );
+
+pub fn maybe_testing() -> Option<Config> {
+    std::env::var("MAYBE_TESTING_MCR").ok().map(|_| Config::default())
```

Comment:
What's the meaning of `MAYBE_TESTING_`?

Reply:
I copy the pattern from the deploy config. It's the way @l-monninger define how to activate optional config part.